templates/search.html

```diff
@@ -128,10 +128,6 @@ <h2 class="icon-heading__title widget__title"><span class="t-short">Find electio
           <h2 class="widget__title"><span class="t-short">So far in the 2016 presidential election:</span></h2>
           <div class="widget__content">
             <div>
-              <h5 class="t-data-header">Candidates running</h5>
-              <span class="t-big-data js-count"></span>
```

Comment:
Is the js code that is using js-count still needed elsewhere, or should it also be removed?